js/templates/mta-sts-result.handlebars

```diff
@@ -0,0 +1,34 @@
+<article class="accordion">
+  <div class="check mta-sts {{status_class}}">
+
+    <div class="accordion-title active js-default-open">
+      <span>MTA-STS</span>
+      <i></i>
+    </div>
+    <div class="accordion-content">
+
+      <p>
+        MTA-STS is a standard that will make it more difficult to perform downgrade attacks on communications with your mailserver.
+      </p>
+
+      {{#if policy}}
+        <p>We found the following{{#ifCond status '===' 0}} valid{{/ifCond}} policy on your mailserver:</p>
+        <pre id="policy-text">{{policy}}</pre>
+
+        {{#ifCond all_messages.length '>' 0}}
+          <h5>Do you manage this email server?</h5>
```

Comment:
Let's add something like, "Here's [a/some] suggestion(s) we have for your deployment:" for warnings and "We found these/this problem(s) with your MTA-STS deployment:" for failures!

Reply:
Great! I wound up adding the framing text to the single message and multiple messages versions, it seemed to work better with the example you sent :)